### DISCUSSION TITLE
Restore from Backup

### DISCUSSION THREAD

**Discussion by maximo1984** [Kareadita/Kavita#3016] [Ideas]
### Idea Description

I had a serious situation where I tried to import the DC Villains month titles which were numbered 23.3, etc. I tried to set them aside as their own volume using the year as the volume (the main titles use 1,2,3 for volume).  What this effectively did on import though, is remove all of the main titles and replace them with the year volume versions. This also erased a considerable number of reading lists I had created. 

I have removed the villain month titles and forced scan the library to bring back the original titles, however the reading and collection lists are now gone. 

How do I restore these from backup?

### Idea Category

User Experience

### Duration of Using Kavita

6 months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by Cducharme84** [comment]
If you have your backup set for nightly runs you could stop kavita, unzip the latest file in your Backups folder into your config folder (or mount if you're using docker) and let it overwrite the files that it asks about. This brings you back to prior to the import when you fire up Kavita again.

There is a caveat, if you had reading progress or made changes to any lists prior to the import but after the backup those would be lost, but it sounds like it'd probably be less loss that way than the current state.

**Reply by maximo1984** [reply]
Do I copy all files in the back up or only the config or a combination ?

**Comment by maximo1984** [comment]
It worked, thanks

**Reply by Cducharme84** [reply]
I was just typing a reply but I'm very glad to hear!
